src/checkstyle/reporter/CodeClimateReporter.hx

```diff
@@ -0,0 +1,56 @@
+package checkstyle.reporter;
+
+import checkstyle.CheckMessage.SeverityLevel;
+import haxe.Json;
+
+class CodeClimateReporter extends BaseReporter {
+
+	static var INFO:String = "info";
```

Comment:
Seems like these are only used in `getSeverity()`, so does it make sense to have them in static vars?